### DISCUSSION TITLE
[New Component] [tree-select-v2] el-tree-select supports virtual scrolling

### DISCUSSION THREAD

**Discussion by TothingWay** [element-plus/element-plus#11961] [Feature Request]
Feature type: **`New Component`**

## Details

- New component name: `el-tree-select-v2`

## Description

I would like to have a component `el-tree-select-v2`

### Features
it combines the functions of components el-tree-v2 and el-select

**Comment by masterworgen** [comment]
+1

**Comment by Ning-Jun** [comment]
有什么替代解决方案吗
